src/cryptocom/exchange/structs.py

```diff
@@ -339,3 +339,74 @@ def create_from_api(
             trigger_price=trigger_price,
             trades=trades
         )
+
+
+@dataclass
+class Interest:
+    loan_id: int
+    currency: str
+    interest: float
+    stake_amount: float
+    interest_rate: float
+
+    @classmethod
+    def create_from_api(cls, data: Dict) -> 'Interest':
+        return cls(
+            loan_id=int(data['loan_id']),
+            currency=data['currency'],
+            interest=float(data['interest']),
+            stake_amount=float(data['stake_amount']),
+            interest_rate=float(data['interest_rate'])
+        )
+
+
+@dataclass
+class Withdrawal:
+    currency: str
+    client_wid: str
+    fee: float
+    create_time: int
+    id: str
+    update_time: int
+    amount: float
+    address: str
+    status: str
+
+    @classmethod
+    def create_from_api(cls, data: Dict) -> 'Withdrawal':
+        return cls(
+            currency=data['currency'],
+            client_wid=data['client_wid'],
+            fee=float(data['fee']),
+            create_time=data['create_time'],
+            id=data['id'],
+            update_time=data['update_time'],
+            amount=float(data['amount']),
+            address=data['address'],
+            status=data['status']
+        )
+
+
+@dataclass
+class Deposit:
+    currency: str
```

Comment:
use coin instead and Coin struct

Reply:
changed as requested, see the last commit